Codeinwp/templates-patterns-collection
# Allow Page Templates Support

### Summary
Allow support to import/export WordPress' Page Templates meta as part of the full-page templates.

### Test instructions
You need to export and import in all editors and see that the page templates are exported/imported properly. Also, make sure to see there are never any errors.

Just to not confuse you, I'm talking about this:

<img width="1061" alt="Screenshot 2021-05-31 at 4 39 47 AM" src="https://user-images.githubusercontent.com/2649903/120123036-4f203f00-c1ca-11eb-832e-b67accd9e283.png">

It can also be accessed in Elementor from here:

<img width="562" alt="Screenshot 2021-05-31 at 4 41 33 AM" src="https://user-images.githubusercontent.com/2649903/120123121-aa523180-c1ca-11eb-9512-3cb507854aea.png">

In Beaver, it'll be the same as Block Template.


Closes #89
